hw/hw13/trigrams.py

```diff
@@ -0,0 +1,87 @@
+import string
+import random
+
+
+def strip_cover(infile):
+    #  strip header and footer, then store the whole book as a string
```

Comment:
Consider using '''Triple quote docstrings'''. See google python style guide for more info: https://google-styleguide.googlecode.com/svn/trunk/pyguide.html#Comments